### PR TITLE
Hpot 108

### PR DESCRIPTION
**Pull request by thatmatt24**
changed container-configuration.yml to just config.yml

created a test_https.py file 

modified generic.py and handler.py for TLS implementation and file name change

added "requests" to the requirements.txt for test_https.py

### CODE REVIEW

File: hpotter-server/hpotter/test/test_https.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import ssl, socket, requests
+
+hostname = '127.0.0.1'
+data = b'test'
+url = 'http://127.0.0.1'
+
+context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+context.load_verify_locations('/Users/mattmcmahon/Desktop/cert.pem')
```
Comment (sheet-t) [inline hpotter-server/hpotter/test/test_https.py]: You have a hard coded path and this is not going to work on anyones elses system. Also we don't have this cert so the test will always fail for everyone other then you

File: hpotter/test/test_tls.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import ssl, socket
+
+hostname = '127.0.0.1'
+# PROTOCOL_TLS_CLIENT requires valid cert chain and hostname
+context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+context.load_verify_locations('/Users/mattmcmahon/Desktop/Matt/HPotter/hpotter/test/cert.pem')
```
Comment (sheet-t) [inline hpotter/test/test_tls.py]: Same hard coded path to cert

File: hpotter/test/test_tls.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ports = [443, 3307, 8090]
+while True:
+    for i in ports:
+        address = (hostname, i)
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM, 0) as sock:
+            with context.wrap_socket(sock, server_hostname=hostname) as ssock:
+                # print(ssock.version())
+                # print(context)
+                # print("sock: ", sock)
+                # print("ssock: ", ssock)
```
Comment (sheet-t) [inline hpotter/test/test_tls.py]: Remove all these print statements unless we need them and please remove the commented out ones unless there is a reason to keep them

File: hpotter-server/hpotter/test/test_https.py (hunk 1 above)
Comment (thatmatt24) [reply hpotter-server/hpotter/test/test_https.py]: haha whoops, I'll move the cert into test/

**Pull request closed without merging**